### PR TITLE
fix #77838 studwork.org

### LINKED ISSUE #77838: studwork.org
### Issue URL (Annoyance)
 [https://studwork.org/spravochnik/oformlenie/cherteji/masshtaby-chertejey-po-gost](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fstudwork.org%2Fspravochnik%2Foformlenie%2Fcherteji%2Fmasshtaby-chertejey-po-gost)
### Comment
Username: `@rakleed`
> 
> --
> The information below is automatically generated. Do not delete it.
> 
> Referrer:
> https://yandex.ru/
> 
> Custom userscripts:
> https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.5.3
Browser: | Chrome
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (180), <br>Hide your Referrer from third-parties (studwork.org), <br>Hide your User-Agent ()
DNS filtering: | server: `https://dns.cloudflare.com/dns-query`<br>
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Russian

### PR DESCRIPTION
**Pull request by wout-codes**
#77838

### CODE REVIEW

File: AnnoyancesFilter/sections/popups.txt [hunk 1]
```diff
@@ -97,6 +97,9 @@ sodapdf.com##.be-tooltip
 !
 ! Specific sites
 !
+studwork.org#$#.base-modal { display: none !important; }
+studwork.org#$#.base-modal__shadow { display: none !important; }
+studwork.org#$#body { overflow: auto !important; }
```
Comment (zloyden) [inline AnnoyancesFilter/sections/popups.txt:102]: studwork.org#$#body { overflow: visible !important; padding-right: 0 !important; }

**Review by zloyden — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/popups.txt (hunk 1 above)
Comment (wout-codes) [reply AnnoyancesFilter/sections/popups.txt:102]: It does the same work. So what's the difference?

Comment (wout-codes) [reply AnnoyancesFilter/sections/popups.txt:102]: And I don't find `padding-right: 0` anywhere.

Comment (zloyden) [reply AnnoyancesFilter/sections/popups.txt]: > So what's the difference?

You can read it [here](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow). I don't see the point in using `auto` here.

> And I don't find `padding-right: 0` anywhere.

When a modal window appears, not only the `overflow` property changes, but also the `padding-right` property.

**Review by zloyden — APPROVED** [review]

**Pull request merged**